source/TradingPanel.cpp

```diff
@@ -95,29 +90,34 @@ void TradingPanel::Step()
 
 void TradingPanel::Draw()
 {
+	const Interface *tradeUi = GameData::Interfaces().Get("trade");
+	const Rectangle box = tradeUi->GetBox("content");
+	const int MIN_X = box.Left();
+	const int FIRST_Y = box.Top();
+
 	const Color &back = *GameData::Colors().Get("faint");
 	int selectedRow = player.MapColoring();
 	if(selectedRow >= 0 && selectedRow < COMMODITY_COUNT)
-		FillShader::Fill(Point(-60., FIRST_Y + 20 * selectedRow + 33), Point(480., 20.), back);
+		FillShader::Fill(Point(MIN_X + 250., FIRST_Y + 20 * selectedRow + 33), Point(480., 20.), back);
```

Comment:
The `Point` being created with literal values should use the content box width minus some border (the default is 20 but that should perhaps be a value stored in a const) as its x value.